google/cloud/storage/oauth2/google_application_default_credentials_file.h

```diff
@@ -23,11 +23,37 @@ namespace cloud {
 namespace storage {
 inline namespace STORAGE_CLIENT_NS {
 namespace oauth2 {
-/// Return the path for the default service account credentials file.
-std::string GoogleApplicationDefaultCredentialsFile();
 
-/// The name of the environment variable to configure `HOME`.
-char const* GoogleApplicationDefaultCredentialsHomeVariable();
+/**
+ * Returns the environment variable that should be checked for a valid file
+ * path when attempting to load Google Application Default Credentials.
+ */
+inline char const* GoogleAdcEnvVar() {
+  static constexpr char kEnvVarName[] = "GOOGLE_APPLICATION_CREDENTIALS";
+  return kEnvVarName;
+}
+
+/**
+ * Returns the path to the file containing Application Default Credentials, if
+ * set in the GOOGLE_APPLICATION_CREDENTIALS environment variable. Returns an
+ * empty string if no such path exists.
+ */
+std::string GoogleAdcFilePathOrEmpty();
+
+/**
+ * Returns the environment variable that should be used to indicate the
```

Comment:
Also consider splitting into a short and longer descriptions.

Reply:
Fixed.